test/python/test_import_boundaries.py

```
@@ -0,0 +1,97 @@
+import sys
+
+import pytest
+
+pytestmark = pytest.mark.L0
+
+_HEAVY = ("torch", "cutlass", "nvidia_cutlass_dsl")
+
+
+def _modules(code: str) -> set:
+    probe = code + "\nimport json, sys; print('@@' + json.dumps(sorted(sys.modules)))"
+    run = subprocess.run([sys.executable, "-c", probe], capture_output=True, text=True)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)